### DISCUSSION TITLE
Getting started documentation?

### DISCUSSION THREAD

**Discussion by RogerTerrazas** [marcusolsson/obsidian-projects#705] [Help]
Is there documentation highlighting the features of the plugin? The medium articles require me to create an account to read.

**Comment by Acylation** [comment]
Hi! Unfortunately we didn't provide a quick start which would be definately useful. If you want to learn how to use this plugin quickly, you can refer to the tutorial videos pinned on the readme. I understand this would take a longer time, but the basic operations are demonstrated well in the videos. Additionally, you can create a demo project when you initialize the project plugin to learn its basic usage. 

<img width="560" alt="image" src="https://github.com/marcusolsson/obsidian-projects/assets/73122375/671db5b2-8bc1-4e26-99ab-25e45e0e5d20">

To initialize the Projects plugin, and there's no important data, you can
- Re-install the plugin, or
- Delete the `data.json` file in the plugin's folder, usually `.obsidian\obsidian-projects\`

Otherwise, please install the plugin in a newly-created vault, or the sandbox vault, to follow the demo project.

**Reply by jkbhagatio** [reply]
Are there plans for the devs to create a user guide?

**Reply by Acylation** [reply]
Thanks for following up! I would say yes but it takes time. I have a tight schedule recently.

**Reply by guhcampos** [reply]
I'd like to reinforce the need for some basic documentation. I've been directed to this plugin by Google, but I have no way of knowing what the plugin does. It's supposed to manage projects, right, but there are hundreds, maybe thousands of project management tools out there, each with its own peculiarities. While trying to find a tool to fit my needs, I need to know what they do first.

I appreciate the work every open source - especially free open source - software developer does, and how you basically donate your time to people you never met. I'm not trying to teach a lesson or whine or anything, just laying out a constructive criticism: most people won't have the time to go through a 45 minute video tutorial just to figure out if a tool fits their needs or not, so it's very likely lots of people like me stumble into this plugin and unfortunatelly move on with their search strictly because of the lack of documentation.

**Reply by quantum-strike** [reply]
Any updates on this? A documentation would be really helpful.

**Reply by Acylation** [reply]
There's a local draft where I played around with docusaurus and plan to set up a github-pages deployment. Though, I need to find time build the contents.

**Comment by Acylation** [comment]
Started up at https://acylation.github.io/obsidian-projects-docs/

Still in construction, will announce when more content being added. Also call for contribution!
